channel.go

```diff
@@ -370,7 +370,10 @@ func (ch *Channel) serve() {
 				if max := 1 * time.Second; acceptBackoff > max {
 					acceptBackoff = max
 				}
-				ch.log.Warnf("accept error: %v; retrying in %v", err, acceptBackoff)
+				ch.log.WithFields(
+					LogField{"error", err.Error()},
```

Comment:
should we make a helper in the package to create a field for an error,
```go
func errorField(err error) LogField {
  return LogField{"error", err.Error()}
}
```

That way we'll have a consistent key and we can't miss calling `err.Error()`